docs/src/pages/[platform]/connected-components/geo/react.mdx

```diff
@@ -121,3 +121,7 @@ with provided `latitude` and `longitude` properties is required.
     import(`./fragments/location-search-standalone.${platform}.mdx`)
   }
 </Fragment>
+
+## Trouble Shooting
```

Comment:
I was thinking more of an Alert "info" variation right below the Quick Start section for Create React App users, rather than a generic callout to the troubleshooting guide.

```
<Alert heading="Create React App setup">
There's a known issue for users of Create React App v5 where the default prod `browserslist` configuration causes build errors when building for production. See the [create react app troubleshooting guide](https://ui.docs.amplify.aws/react/getting-started)./troubleshooting#create-react-app) to configure Create React App for use with the MapView UI component.
</Alert>
```

Reply:
done e01b5d221